### ISSUE TITLE
WCAG: organisation select-element name

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2180]
The select element in the left column under the label "Organization" has no accessible name. The label is not linked to the select element by means of a "for and id" construction.

The reading software cannot tell its user what the subject of this select element is.

In the code, the fields must be linked to labels via an ID, so that it is clear to auxiliary software what needs to be entered where. 

**Dutch:**
Het select-element in de linker kolom onder het label ‘Organisatie’ heeft geen toegankelijke naam. Het label is niet verbonden met het select-element door middel van een ‘for en id’ constructie.

De voorleessoftware kan niet aan zijn gebruiker vertellen wat het onderwerp is van dit select-element.

In de code moeten de velden gelinkt worden aan labels via een id, zodat het voor hulpsoftware duidelijk is wat er waar ingevoerd moet worden.

**Comment by SjoerdHilhorst** [comment]
gefixt

**Comment by maxvisser** [comment]
Was about 'Organisatie' filter on products page which was renamed to 'Aanbieders'

![image](https://user-images.githubusercontent.com/10818702/122244586-49fb0980-cec5-11eb-9b6f-97531edc7d23.png)
